dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+1. What if some AdUnit auctions should be skipped entirely for some devices?
+1. What if some bid requests apply only to users originating from certain countries? 
+
+By supporting these scenarios, header bidding can be more efficient - the browser can send bids to a more surgical set of bidders based on device size or other attributes the page code can create.
+
+Before going into how these scenarios work, a few words about the design of Prebid's approach. With Prebid 1.0,
+we introduce 'labels', which are just generic tags. Labels are defined by code in the page and then used by certain AdUnits and bids.
+
+The basic steps are:
+
+1. Build up an array of labels from two sources: either as an output of sizeConfig or as an optional argument to requestBids().
+1. Apply label targeting to AdUnits or specific bids.
+
+Let's jump into an example.
+
+## Use Case: What if some bidders should be skipped for some devices?
+
+Say a particular bidder is focused on mobile phone demand, so it's really not worthwhile for either party
+to send them requests from display or tablets.
+
+We'll start with how to set up the labels from sizeConfig:
+
+{% highlight js %}
+
+pbjs.setConfig({
+  sizeConfig: [{
+       mediaQuery: '(min-width: 1200px)',
+       sizesSupported: [[970,90], [728,90], [300,250], [160,600]],
+       label: [ "display"]
+     }, {
+       mediaQuery: '(min-width: 768px) and (max-width: 1199px)',
+       sizesSupported: [[468,60], [300,250], [160,600]],
+       label: [ "tablet"]
+     }, {
+       mediaQuery: '(min-width: 0px) and (max-width: 767px)',
+       sizesSupported: [[300,50],[320,50],[120,20],[168,28]],
+       label: [ "phone"]
+  }]
+});
+
+{% endhighlight %}
+
+Here, labels are applied for each of the 3 screen sizes that can later be used in
+conditional AdUnit logic. For example:
+
+{% highlight js %}
+
+var AdUnits = [{
+    code: "ad-slot-1",
+    sizes: [[768,90], [468,60], [320,50]], 
+    bids: [
+        {
+            bidder: "bidderA",
+            params: {
+                placement: "1000"
+            }
+       },{
+            bidder: "mobileBidder",
+            labelAny: ["phone"],  // this bid only applies to small screensizes
+            params: {
+                placement: "2000"
+            }
+       }
+   ]
+}]
+
+{% endhighlight %}
+
+How this works:
+
+1. Users with a screen width of 767px and under will cause the third mediaQuery to fire.
+1. This rule puts "phone" into the label array and a list of sizes into sizesSupported
+1. Then the AdUnit is processed:
+    1. The first bid doesn't have any conditional logic, so is present in every auction.
+    1. The second bid requires that "phone" be present in the label array, otherwise it won't be part of the auction.
+
+## Use case: What if some bidders have different parameters for different devices?
+
+For reporting and targeting purposes, Publishers and SSPs sometimes break out different 'placements' for different platforms.
```

Comment:
Can you write more about the quotes around placements?  Do you mean to imply that "placement" is a new term for the reader?  If so we should define it briefly.

Or do you mean that it can mean a few different things depending on context?  If so, can we explicitly share what some of those are?